<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QDialog" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>client</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2" colspan="2">
    <widget class="QLabel" name="labelServerName">
     <property name="text">
      <string>Server:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QComboBox" name="serverName"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="labelServerPort">
     <property name="text">
      <string>Server port:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLineEdit" name="serverPort"/>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>This app requires that you run the ScorpionServer example as well.</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QTextEdit" name="conversation">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="sendButton">
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="connectButton">
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QPushButton" name="exitButton">
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" rowspan="3" colspan="3">
    <widget class="QLineEdit" name="messageLine"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
